CHANGELOG.md

```diff
@@ -1,3 +1,53 @@
+# v0.2.0 dbt_qualtrics_source
+
+[PR #5](https://github.com/fivetran/dbt_qualtrics_source/pull/5) includes the following updates: 
+
+## 🚨 Breaking Changes: Bug Fixes 🚨
+- Casted the following timestamp fields in the below models using the `dbt.type_timestamp()` macro. This is necessary to ensure all timestamps are consistently casted and do not experience datatype mismatches in downstream transformations.
+    - stg_qualtrics__contact_mailing_list_membership
+        - unsubscribed_at
+    - stg_qualtrics__directory_contact
+        - created_at
+        - unsubscribed_from_directory_at
+        - last_modified_at
+    - stg_qualtrics__directory_mailing_list
+        - created_at
+        - last_modified_at
+    - stg_qualtrics__distribution_contact
+        - opened_at
+        - response_completed_at
+        - response_started_at
+        - sent_at
+    - stg_qualtrics__distribution
+        - created_at
+        - last_modified_at
+        - send_at
+        - survey_link_expires_at
+    - stg_qualtrics__survey_response
+        - finished_at
+        - is_finished
```

Comment:
you can remove `is_finished`, it was not updated.

Reply:
Not sure why that snuck in there. Probably an artifact of copy/paste. Thanks for catching!

Reply:
Updated